salt/modules/test_virtual.py

```diff
@@ -1,6 +1,7 @@
-"""
-Module for running arbitrary tests with a __virtual__ function
-"""
+'''
```

Comment:
Pre-commit didn't ran locally?
These should be double quotes ...

Reply:
I actually have no clue how they changed - the only thing I changed was the contents of the docstring itself :confused: